src/UserAdmin/src/Smart.FA.Catalog.Web/Authentication/Header/Headers.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Smart.FA.Catalog.Web.Authentication.Header;
+
+public static class Headers
+{
+    public const string UserId = "userId";
+    public const string ApplicationName = "smartApplication";
+    public const string AccountData = "customData";
```

Comment:
Why not call it `accountData`?
```suggestion
    public const string AccountData = "accountData";
```

Reply:
As mentionned the name of the header cannot be changed. It's part of the library created by Anthony. It's the general name to pass specific data that are not part of the default header (like userid and smartapplication)